pkg/build/node/node.go

```diff
@@ -308,7 +308,19 @@ func (c *BuildContext) buildImage(dir string) error {
 	}
 
```

Comment:
We should enable containerd service before this, shouldn´t we?
`systemctl enable containerd`

Reply:
we _did_ need to enable containerd but _not_ for this. we run containerd one-off during the node image build, not under systemd. less moving parts, in the future we'll only do the image import using the API directly